lib/CleantalkSP/SpbctWP/DB/SQLSchema.php

```diff
@@ -51,6 +51,10 @@ class SQLSchema extends \CleantalkSP\Common\DB\SQLSchema {
                 array('field'=>'source_status',  'type' => 'set("UP_TO_DATE","OUTDATED","NOT_IN_DIRECTORY","UNKNOWN")', 'null' => 'yes', 'default' => 'NULL',),
                 array('field'=>'version',        'type' => 'varchar(20)',   'null' => 'yes', 'default' => 'NULL',),
                 array('field'=>'checked',        'type' => 'enum("NO", "YES", "YES_SIGNATURE", "YES_HEURISTIC")', 'null' => 'no', 'default' => '"NO"',),
+                array('field'=>
+                          'checked_heuristic',   'type' => 'int', 'null' => 'no', 'default' => '0',),
```

Comment:
Please, add the size to the column 'int(1)'

Reply:
If you add the size to the int type, there will be a warning, you do not need to add the size to the number.

warning: 1681 Integer display width is deprecated and will be removed in a future release.